card/mod/01_core/set/all/name.rb

```diff
@@ -285,7 +239,7 @@ def suspend_name name
   Card.where(id: id).update_all(name: tmp_name, key: tmp_name)
 end
 
-event :cascade_name_changes, after: :store, on: :update, changed: :name do
+event :cascade_name_changes, :finalize, on: :update, changed: :name do
```

Comment:
the comment in the code below suggests we intentionally avoid events on all the cascaded cards.  At some point we should discuss whether that's still a good idea.  Perhaps the new API changes things?